src/main/java/roomescape/RoomescapeController.java

```diff
@@ -0,0 +1,15 @@
+package roomescape;
+
+import org.springframework.stereotype.Controller;
+import org.springframework.web.bind.annotation.GetMapping;
+import org.springframework.web.bind.annotation.RequestMapping;
+
+@Controller
+@RequestMapping("/admin")
+public class RoomescapeController {
+
+    @GetMapping()
```

Comment:
`()` 부분은 생략해주셔도 됩니다! 😃 

```suggestion
    @GetMapping
```

Reply:
넵!! 감사합니다!!😁